corehq/apps/dump_reload/sql/dump.py

```diff
@@ -7,40 +7,79 @@
+    FilteredModelIteratorBuilder('sms.PhoneNumber', SimpleFilter('domain')),
+    FilteredModelIteratorBuilder('scheduling.AlertSchedule', SimpleFilter('domain')),
+    FilteredModelIteratorBuilder('scheduling.AlertEvent', SimpleFilter('schedule__domain')),
+    FilteredModelIteratorBuilder('scheduling.ImmediateBroadcast', SimpleFilter('domain')),
+    FilteredModelIteratorBuilder('scheduling.TimedSchedule', SimpleFilter('domain')),
+    FilteredModelIteratorBuilder('scheduling.TimedEvent', SimpleFilter('schedule__domain')),
+    FilteredModelIteratorBuilder('scheduling.ScheduledBroadcast', SimpleFilter('domain')),
+    FilteredModelIteratorBuilder('scheduling.AlertScheduleInstance', SimpleFilter('domain')),
+    FilteredModelIteratorBuilder('scheduling.CaseAlertScheduleInstance', SimpleFilter('domain')),
+    FilteredModelIteratorBuilder('scheduling.CaseTimedScheduleInstance', SimpleFilter('domain')),
+    FilteredModelIteratorBuilder('scheduling.TimedScheduleInstance', SimpleFilter('domain')),
+    UniqueFilteredModelIteratorBuilder('scheduling.SMSContent', SimpleFilter('alertevent__schedule__domain')),
+    UniqueFilteredModelIteratorBuilder('scheduling.SMSContent', SimpleFilter('timedevent__schedule__domain')),
+    UniqueFilteredModelIteratorBuilder('scheduling.EmailContent', SimpleFilter('alertevent__schedule__domain')),
+    UniqueFilteredModelIteratorBuilder('scheduling.EmailContent', SimpleFilter('timedevent__schedule__domain')),
+    UniqueFilteredModelIteratorBuilder('scheduling.SMSSurveyContent', SimpleFilter('alertevent__schedule__domain')),
+    UniqueFilteredModelIteratorBuilder('scheduling.SMSSurveyContent', SimpleFilter('timedevent__schedule__domain')),
+    UniqueFilteredModelIteratorBuilder('scheduling.IVRSurveyContent', SimpleFilter('alertevent__schedule__domain')),
+    UniqueFilteredModelIteratorBuilder('scheduling.IVRSurveyContent', SimpleFilter('timedevent__schedule__domain')),
+    UniqueFilteredModelIteratorBuilder('scheduling.CustomContent', SimpleFilter('alertevent__schedule__domain')),
+    UniqueFilteredModelIteratorBuilder('scheduling.CustomContent', SimpleFilter('timedevent__schedule__domain')),
+    FilteredModelIteratorBuilder('sms.SMS', SimpleFilter('domain')),
+    FilteredModelIteratorBuilder('sms.Keyword', SimpleFilter('domain')),
+    FilteredModelIteratorBuilder('sms.KeywordAction', SimpleFilter('domain')),
+    FilteredModelIteratorBuilder('sms.QueuedSMS', SimpleFilter('domain')),
+    FilteredModelIteratorBuilder('sms.SelfRegistrationInvitation', SimpleFilter('domain')),
+    FilteredModelIteratorBuilder('smsbillables.SmsBillable', SimpleFilter('domain')),
+    UnfilteredModelIteratorBuilder('smsbillables.SmsGatewayFee'),
+    UnfilteredModelIteratorBuilder('smsbillables.SmsGatewayFeeCriteria'),
+    FilteredModelIteratorBuilder('smsbillables.SmsUsageFee', SimpleFilter('criteria__domain')),
+    FilteredModelIteratorBuilder('smsbillables.SmsUsageFeeCriteria', SimpleFilter('domain')),
```

Comment:
i'd say we can probably leave out the smsbillables models for now. it's not used for enikshay and some of these models get created automatically on first install which will probably make matching foreign keys up a challenge